Made it possible to paint windows to make them opaque.

xxalpha opened the pull request:
Edit: Changed it so only darker colors make the window opaque.
Changed only the colour var.
Painting or cleaning tinted windows will not make them see through.

-------------

My first PR.

![scrnshot1](https://cloud.githubusercontent.com/assets/4951579/10859780/b7365602-7f58-11e5-97ef-39150fa9a3b6.png)

Paint them with spray cans, clean them with soap.

Also renamed two vars to be less confusing.

francinum commented:
>Color instead of Colour
:-1:

Anonus commented:
Should work with paint cans too.
>the color/colour wars
stop it

RemieRichards commented:
changing ```colourName``` to ```colorName``` was done 1000% needless.

xxalpha commented:
I wouldn't call consistency needless.

RemieRichards commented:
We've been over this, we're not doing the whole "Unified spelling" change.
We're continuing as we always have, Developers write in what they prefer and DON'T NEEDLESSLY CHANGE THINGS.

xxalpha commented:
I am not scouring the codebase for 'colour', I changed the two vars from the same type of object because one was a bad name. I don't write 'colour', so it would have 'paint_color' and 'colourName', so I just changed both.

RemieRichards commented:
You could have just as easily written ```paint_colour``` then you'd have had your sweet "consistency" with LESS WORK TO DO.

xxalpha commented:
Stop bullying my PR, remie.

GunHog commented:
Inconsistent spellings are a consistency and maintainability issue. US or UK spelling, I suggest we pick one and stick with it, or this problem just gets worse.

RemieRichards commented:
We've dealt with it for 5 years, it's literally a non issue.
Just stop changing shit you didn't write FOR NO PURPOSE.

xxalpha commented:
Not compatible with brits.

Anonus commented:
Do you mind if I just take this and do it myself?

Cheridan commented:
omg stop nerding out over fucking spelling who cares either way

duncathan commented:
@RemieRichards look it doesn't matter which spelling is used and if this PR were just changing it for the sake of making it "correct" I'd be against it but
`colour` as a var name is disgusting and confusing; `paint_color` is perfectly fine and descriptive

RemieRichards commented:
@duncathan Omg, I'm not defending ```colour```, I'm having a go at xxalpha for renaming a completely seperate var, rather than just having his new var name be ```paint_colour``` instead of ```paint_color```+```colorName```

tl;dr he needlessly changed 2 vars vs. changing 1 var.

duncathan commented:
@RemieRichards ohh I see your point
Well for this I'll just say "it's not really a big enough deal to make a fuss about"
I will say that given the fact that `color` is used internally it'd be nice to at the very least maintain consistency with that but again, doesn't really matter too much

Anonus commented on code/game/objects/items/weapons/clown_items.dm:
```diff
@@ -66,6 +66,10 @@
+	else if(istype(target, /obj/structure/window))
+		user << "<span class='notice'>You clean \the [target.name].</span>"
+		target.color = initial(target.color)
+		target.SetOpacity(0)
```
This will cause tinted windows to become non-opaque, you need to check if it's tinted.

xxalpha replied:
Tinted windows aren't used.

francinum replied:
@xxalpha Yes they are.

duncathan replied:
They shouldn't be tbh

francinum replied:
They are, mostly for the chapel confession rooms, but some mapper probably used them elsewhere. Anyhow, that's not an excuse to not add a check for them.

Saegrimr replied:
Tinted windows are in box maint as well below toxins. It's been there for ages if anybody here has even been in maint in the last 2 years.

WJohn commented:
:+1: 
Cute.

Jordie0608 commented:
Painting windows is cool, opaque windows aren't.

Niknakflak commented:
Watching remie freak out over the color thing again is funny.  If it doesn't "matter", who cares?

optimumtact commented:
I like the idea, @Jordie0608 what is so wrong with opaque windows?

Niknakflak commented:
Watching remie freak out over the color thing again is funny. If it doesn't "matter", who cares?

optimumtact commented:
Taking passive aggressive potshots against people isn't funny. Keep your little back and forth off github please.

CosmicScientist commented:
Painting windows, :+1:.

Lumping it together with changing var names that apply to more than the title of this PR, :-1:.

ike709 commented:
Window painting: :+1: 

Reenacting the War of 1812: :-1:
